app/virtual_gamepad_hub.js

```diff
@@ -44,7 +44,8 @@ Virtual gamepad hub class
         this.gamepads[padId] = new gamepad();
         return this.gamepads[padId].connect(function() {
           return callback(padId);
-        }, function(err) {
+        }, (err)=> {
+          this.gamepads[padId] = void 0;
```

Comment:
Changing things in the server's JS code won't do much as it's written in coffee script and the JS code is generated from it. I will take care of an equivalent change in the coffee script.